### DISCUSSION TITLE
1.3.1

### DISCUSSION THREAD

**Discussion by Carseason** [Carseason/routerdog#38] [Announcements]
1. 移除 gouci 依赖，使用 cmd uci 来实现，修复 下载中心 读取 aria2 配置时遇到空 list 异常问题
2. 替换 iphtos 预览照片组件为 https://github.com/Carseason/vue3-photo-preview , 支持 pc 移动端 和 照片追加
3. 修复 iphtos 切换页面时的路由缓存问题

**Comment by pandamelive** [comment]
越来越强啦

**Comment by newqiang** [comment]
iphotos怎么设置存储路径，点击没反应

**Reply by Carseason** [reply]
主页、Cloud

**Reply by newqiang** [reply]
谢谢，设置好了

**Comment by Fengerxuanxiao** [comment]
出现Bug，下载中心清楚任务后，已完成和停止不会清零

**Reply by Fengerxuanxiao** [reply]
文件管理能否像Windows电脑那样固定文件夹到快速访问

**Comment by tizips** [comment]
大佬，目前每次重启，自定义的桌面图标都会消失，能优化下吗

**Reply by Carseason** [reply]
主页 、Cloud 设置存储路径

**Reply by tizips** [reply]
3Q，解决了
